packages/squiggle-lang/src/fr/number.ts

```diff
@@ -20,9 +26,7 @@ function makeNumberArrayToNumberDefinition(
   throwIfEmpty = true
 ) {
   return makeDefinition([frArray(frNumber)], ([arr]) => {
-    if (arr.length === 0 && throwIfEmpty) {
-      return throwEmptyList();
-    }
+    throwIfEmpty && assertIsNotEmpty(arr);
```

Comment:
`makeNumberArrayToNumberArrayDefinition` still uses `throwEmptyList`; do the same change there and remove then `throwEmptyList`?

Reply:
Good point, fixed!